cogs/cog_todolist.py

```diff
@@ -0,0 +1,45 @@
+import discord
+from discord.ext import commands
+from utils import config as cfg
+from utils import todolist_priority_queue
+from models import todolist
+
+
+class ToDoListCommands(commands.Cog, name="ToDoList Commands"):
+    """These are the notes commands"""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="create-item", aliases=["ci"])  # command to create a note
+    async def create_item(self, ctx, time: str, *msg: str):
+        userid = ctx.author.id
+
+        # Timer creation
+        msg = ' '.join(msg)
+        item = todolist.ToDoList(userid=userid, td_secs=int(time) * 60, msg=msg, discord_message=ctx.message)
+
+        # P-queue and database update
+        todolist_priority_queue.ToDoListPriorityQueue.get_instance().add_task(item)
+        # item.insert(['message_id', 'userid', 'channel_id', 'start_time', 'end_time', 'msg'])
+
+        await ctx.send(embed=discord.Embed(
+            description="ToDoList Item created!\n",
+            colour=cfg.colors.SUCCESS
+        ))
```

Comment:
We should get to making a good EmbedBuilder class to better handle these hanging embeds.